audit/.support.sh

```diff
@@ -36,6 +36,6 @@ assert_audit_file_content() {
 }
 
 set_lb_cookie() {
-  local amlbcookie=$([ $TEST_INSTANCE_ID == 2 ] && echo "03" || echo "0$TEST_INSTANCE_ID")
+  local amlbcookie=$([ $TEST_INSTANCE_ID = 2 ] && echo "03" || echo "0$TEST_INSTANCE_ID")
```

Comment:
This contains incorrect logic that assumes cookie value can be mapped to instance ID.

Reply:
I have used associative array of AM instances and their respective LB cookie values instead.